backend/app/notify/email.go

```diff
@@ -164,35 +165,55 @@ func (e *Email) setTemplates() error {
 	return nil
 }
 
-// Send email about comment reply to Request.Email if it's set,
-// also sends email to site administrator if appropriate option is set.
+// Send email about comment reply to Request.Emails and Request.AdminEmails
+// if they're set.
 // Thread safe
 func (e *Email) Send(ctx context.Context, req Request) error {
-	if req.Email == "" {
-		// this means we can't send this request via Email
-		return nil
-	}
 	select {
 	case <-ctx.Done():
-		return errors.Errorf("sending message to %q aborted due to canceled context", req.Email)
+		return errors.Errorf("sending email messages about comment %q aborted due to canceled context", req.Comment.ID)
 	default:
 	}
 
-	if req.parent.User.ID == req.Comment.User.ID && !req.ForAdmin {
-		// don't send anything if if user replied to their own comment
-		return nil
+	result := new(multierror.Error)
+
+	// send user notifications
+	for _, email := range req.Emails {
+		email := email
+		log.Printf("[DEBUG] send user notification via %s, comment id %s", e, req.Comment.ID)
+		msg, err := e.buildMessageFromRequest(req, email, false)
+		if err != nil {
+			return err
+		}
+
+		err = repeater.NewDefault(5, time.Millisecond*250).Do(
+			ctx,
+			func() error {
+				return e.sendMessage(emailMessage{from: e.From, to: email, message: msg})
+			})
+
+		result = multierror.Append(errors.Wrapf(err, "problem sending user email notification to %q", email))
 	}
-	log.Printf("[DEBUG] send notification via %s, comment id %s", e, req.Comment.ID)
-	msg, err := e.buildMessageFromRequest(req, req.ForAdmin)
-	if err != nil {
-		return err
+
+	// send admin notifications
+	for _, email := range req.AdminEmails {
+		email := email
+		log.Printf("[DEBUG] send admin notification via %s, comment id %s", e, req.Comment.ID)
```

Comment:
two blocks seens to be identical, except the last param (forAdmin). Maybe extract to an inner func?